spec/passaporte_web/notification_spec.rb

```diff
@@ -170,7 +170,7 @@
         it "should return false and set the errors with the reason for the failure, authenticated as the application" do
           notification.destination = nil # required field
           notification.should_not be_persisted
-          notification.save('application').should be_false
+          expect(notification.save('application')).to be_falsy
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.